ktor-server/ktor-server-plugins/ktor-server-openapi/jvm/src/io/ktor/server/plugins/openapi/OpenAPI.kt

```diff
@@ -19,16 +19,17 @@ import java.io.*
  * The documentation is generated using [StaticHtml2Codegen] by default. It can be customized using config in [block].
  * See [OpenAPIConfig] for more details.
  */
-public fun Routing.openAPI(
+public fun Route.openAPI(
     path: String,
     swaggerFile: String = "openapi/documentation.yaml",
     block: OpenAPIConfig.() -> Unit = {}
 ) {
-    val file = resolveOpenAPIFile(swaggerFile)
+    val api = resolveOpenAPIFileContent(swaggerFile)
```

Comment:
In the OpenAPI specification, they call it a document, so `openAPIDocument` or `document` would be a better name for the variable.